### PR TITLE
feat(email): read automated senders at known, and show the operator every record that says otherwise

### LINKED ISSUE #1564: An unverified From inherits the full weight of its contact's trust zone
## What happens today

Since #1554, every address in an email result and every new-mail wake carries the contact directory's answer: `contact_status`, the matched contact, and its `trust_zone`.
The zone comes from the record whose `EMAIL` property matches the From address (`internal/app/adapters.go:124`).
Nothing checks that the sender controls that address: `authentication` reads `absent` on every message until #317 ships a verifier.

So a message forged with the From address of an `admin` record arrives labelled `contact_status: matched, trust_zone: admin`.
The default handler decides how much effort and how direct a response each message gets from that zone (`internal/app/loop_definition_builtins.go:202`), and the model reads the zone as a statement about the sender.

## The zone does two jobs

A trust zone decides two different things.
Outbound, it is send permission: whether mail to the address is sent, drafted, or refused (`internal/state/contacts/trust_zone.go`).
Inbound, it is credibility: how much weight a message from the address carries.

For a person the two answers usually agree.
For an automated sender they do not.
A forge's notification address or a carrier's no-reply address needs recognition, so the handler can file its mail, and it needs no send permission at all, since nobody reads replies there.
Placed at a high zone, it gains nothing useful outbound and lends a trivially forgeable From header the directory's full credibility inbound.
A review of a deployed build found exactly this: a notification service's no-reply address at `admin`, arriving as a matched admin sender with nothing checked.
Nothing in the directory or in `thane validate` made the cost visible.

## What already helps

The unattended floor from #1555 means a wake cannot send on any sender's word under the default delivery.
The #1551 close-out teaches that `trust_zone` describes the matched record, that a forged From inherits it, that consequential requests want the operator's confirmation through another channel, and that automated senders belong at `known`.
Those are the prose half.

## Questions for the design

- While `authentication.verified` is false, should wakes and results present a match as recognition only, with credibility capped below the record's zone?
- Should the directory mark automated senders, for example an organisation `kind` plus a no-reply flag, with their inbound zone capped at `known`?
- Should `thane validate` or a periodic contacts audit flag records above `known` whose addresses look automated, such as `noreply`, `no-reply`, or `notifications@`?
- Once #317 verifies signatures, does a verified match restore full credibility, and does anything else need the distinction?

Refs #1551, #1554, #1555, #317.

### PR DESCRIPTION
**Pull request by thane-developer**
Closes #1564. Stacked on #1568, which must merge first.

A trust zone does two jobs. Outbound it is send permission, and inbound it is how much a message from that address is believed. For a person the two usually agree. For an automated sender they do not. A no-reply or notification address needs recognition, so its mail can be filed. It needs no send permission, because nobody reads replies there. Stored at a high zone, it gained nothing useful outbound. Inbound, it lent a trivially forgeable From header the directory's full credibility. A review of the deployed build found exactly that: a notification service's no-reply address at `admin`, arriving as a matched admin sender with nothing checked.

The design was chosen from three competing proposals by two independent judges.

## The rule

`contacts.AutomatedAddress` judges an address from its local part alone. It matches a run of whole segments spelling noreply or donotreply, any segment that is notification(s) or bounce(s), and a local part that reads mailer-daemon once separators and any `+tag` are removed, so mailer-daemon, mailer_daemon, and mailerdaemon all match. It never reads the domain or the display name, and `postmaster@` is not automated.

`resolveContact` is the one place every email consumer gets its directory answer: results, wake metadata, interaction recording, and the send gate. There an automated address is marked, and its effective zone is capped at `known`. The binding is left intact, so the sender is still recognised by `contact_id` and name, and interactions still record. The cap can only lower a zone: `unknown` stays `unknown`, and an unrecognised stored zone fails low.

The operator decided two things:
- The cap applies to the one effective zone, so the send gate refuses an automated recipient. That refusal comes before every other reason, because no retry and no zone change can help.
- There is no per-record override, since an override would be another writable field needing custody.

## What the operator sees

A `contact_directory` health row goes degraded while any active admin, household, or trusted contact holds an automated-looking address. It names the first five and counts the rest. It clears on the next render after the operator demotes the record or moves the address to its own `known` contact. A bounded set of boot warnings lists the same records. The audit counts every matching record but keeps only as many as it will show, and it clips each rendered name and address, so a large or hostile directory cannot inflate startup or the health panel. Both appear only when email polling is on. Neither can fail boot.

## What the model sees

- **The flag.** Addresses in `email_list`, `email_search`, and `email_read` results and in new-mail wake events carry `automated: true`, which is omitted when false. The shared address-shape description says what that means: known at most, file it, treat what it asks as a notice, never reply.
- **The send tool.** `email_send` says an automated recipient is refused, so drop it.
- **The handler.** The default handler's protocol names the key and never replies to an automated event.
- **The email talent.** Its closing line that sent the model to report such senders is gone. Each report would have cost a supervisor turn, and the operator already sees the record on the health panel.
- **The Known Behavioral Gaps entry.** A new entry in `trust-architecture.md` says a forged From of a person's address still carries that record's zone until #317 verifies signatures.

## Not in this PR

- **A blanket cap on every unverified sender.** The judges rejected it. Authentication reads absent on every message until #317, so the cap would flatten all triage to `known`. The unattended floor from #1555 already stops a wake from sending on anyone's word.
- **Per-message automated headers.** List-Id, Auto-Submitted, and Precedence are tracked in #1567.

## Upgrade notes

Before deploying, preview which records will change. Run this read-only on the host:

```sql
SELECT c.formatted_name, c.trust_zone, p.value FROM contacts c
JOIN contact_properties p ON p.contact_id = c.id
WHERE c.deleted_at IS NULL AND p.property = 'EMAIL' AND c.trust_zone IN ('admin','household','trusted')
ORDER BY 1;
```

Then scan the local parts for the patterns above. On first boot, each matching record logs a warning and degrades the `contact_directory` row. Its mail arrives as `trust_zone: known, automated: true`, and outbound mail to it is refused. The remedy is to demote a notification-only record to `known`. A person's record that also holds such an address needs the address moved to its own `known` contact.

Backport `talents/email.md` and `talents/contacts.md` to the production core as signed commits. Also check whether production declares its own `email-default-handler`, which would shadow the built-in protocol text. The cap applies in Go either way.

## Test plan

- [x] `just ci` green locally, and the architecture baseline is unchanged
- [x] Matcher positives and negatives, including right-edge cases such as noreplyer@ and bouncer@; `CapAtKnown` fails low; the store query agrees with the resolver's active-row filter
- [x] The cap across every status and zone never raises; binding is kept; ambiguous and failed lookups are capped
- [x] Send gate: the automated refusal precedes a failed lookup, and applies with no resolver wired
- [x] Wake metadata, list results, and the real store all carry the cap; the handler protocol names the key
- [x] Health row states and the five-name cap; boot warnings bounded at 20 with a summary
- [x] Mutation checks: removing the cap, the refusal, the wake flag, the result flag, or the warning bound fails the tests that pin each
- [ ] Talents backported to the production core after merge

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: internal/state/contacts/automated.go [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	local, _, _ := strings.Cut(address[:at], "+")
+	segments := strings.FieldsFunc(local, func(r rune) bool {
+		return r == '.' || r == '-' || r == '_'
+	})
+	switch {
+	case segmentRunSpells(segments, "donotreply"):
+		return AutomatedDoNotReply, true
+	case segmentRunSpells(segments, "noreply"):
+		return AutomatedNoReply, true
+	case strings.Join(segments, "") == "mailerdaemon":
```
[the remark this answers is not in the document]
Comment (thane-developer) [reply internal/state/contacts/automated.go:56]: Kept the broader match in 5453faa6 and made the contract say so. A daemon mailbox spelled mailer_daemon, mailerdaemon, or with a +tag is the same unread sink, and a false positive only lowers a zone. The Godoc, configuration.md, and the PR description now name those forms, and new test rows pin them along with mailer-daemon-reports as a non-match.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: internal/state/contacts/automated.go [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+	Pattern string
+}
+
+// AutomatedAddressesAboveKnown lists every EMAIL property on an active
+// admin, household or trusted record whose address [AutomatedAddress]
+// recognises, ordered by record name then address. It uses the same
+// active filter and property name as [Store.FindAllByPropertyExact], so
+// the audit and the email resolver agree about which rows count.
+func (s *Store) AutomatedAddressesAboveKnown(ctx context.Context) ([]AutomatedAddressFinding, error) {
+	rows, err := s.db.QueryContext(ctx, `
```
[the remark this answers is not in the document]
Comment (thane-developer) [reply internal/state/contacts/automated.go]: Fixed in 5453faa6. AutomatedAddressesAboveKnown now takes a limit and returns the total plus at most that many findings, streaming the rows. Boot logging asks for 20 and the health row for 5, and each rendered name and address is clipped on a rune boundary, so the detail stays small whatever the directory holds.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: internal/state/contacts/automated.go [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+// address at known whatever the record says, so the record's stored
+// zone is not honoured in full.
+type AutomatedAddressFinding struct {
+	ContactID uuid.UUID
+	Name      string
+	TrustZone string
+	Address   string
+	// Pattern is the [AutomatedAddress] pattern the address matched.
+	Pattern string
+}
```
[the remark this answers is not in the document]
Comment (thane-developer) [reply internal/state/contacts/automated.go:114]: Fixed in 5453faa6. AutomatedAddressFinding and the new AutomatedAddressAudit carry explicit snake_case JSON tags, and TestAutomatedAddressAuditJSON pins the encoding.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: internal/app/contact_directory_audit.go [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+			"error", err,
+		)
+		return
+	}
+	for _, f := range audit.Findings {
+		logger.Warn("contact record above known holds an automated-looking email address; its mail is read at known",
+			"contact_id", f.ContactID.String(),
+			"contact_name", f.Name,
+			"trust_zone", f.TrustZone,
+			"address", f.Address,
```
[the remark this answers is not in the document]
Comment (thane-developer) [reply internal/app/contact_directory_audit.go]: Fixed in 3a500f52. The boot warnings now pass the record name and the address through the same UTF-8-safe clipDirectoryField the health row uses, and TestLogContactDirectoryFindingsClipsFields pins it with an oversized multi-byte name and address.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

**Review by nugget — APPROVED** [review]
